package.json

```diff
@@ -23,8 +23,7 @@
     "build": "npm run build:data && npm run build-css && next build",
     "build-css": "node-sass --include-path scss styles/index.scss static/index.css --output-style compressed && node-sass --include-path scss styles/success-banner.scss static/success-banner.css --output-style compressed",
     "serve": "next start",
-    "export": "npm run build && next export -o build && cp CNAME build/CNAME && cp .nojekyll build/.nojekyll && npm run copy-static",
-    "copy-static": "cp -r static_html/* build/",
+    "export": "npm run build && next export -o build && cp CNAME build/CNAME && cp .nojekyll build/.nojekyll",
```

Comment:
I'll mention it, since it was introduced to recently (yesterday) I don't think we need another commit.